packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/InputElements/OptionTypes/AddOption.tsx

```diff
@@ -29,6 +30,11 @@ export const AddOption: React.FC<AddOptionProps> = ({
       >
         Add Option
       </Button>
+      <UploadImage 
+        onImageUpload={(markdownUrl) => {
+          addOption([makeTag(6), "!" + markdownUrl], choices, callback);
```

Comment:
we can avoid an unnecassary ! every time if we jjust move it inside the formatImageUrl ?